src/hotspot/share/logging/logOutputList.hpp

```diff
@@ -48,11 +49,11 @@ class LogOutputList {
  private:
   struct LogOutputNode : public CHeapObj<mtLogging> {
     LogOutput*      _value;
-    LogOutputNode*  _next;
+    LogOutputNode* volatile _next;
```

Comment:
Atomic::load_acquire will cast its argument to const volatile T* no matter what.
 
another reason is that there's no other load of _next field. I don't see any chance compiler elides load instruction for it. 
so I don't think we need 'volatile' qualifier for it. same reasons for  _level_start[LogLevel::Count]

Reply:
I have referred to the usage of Atomic::load_acquire in hotspot, and found that most of them use 'volatile' qualifier even if there is no multiple load of the field in some situation. Furthermore, I think adding volatile will not cause performance or correctness issue (harmless), thus I tend to add it.

Reply:
Using a volatile qualifier for atomically-accessed fields is standard practice in HotSpot. Apart from anything else, it's good for readability.